src/app/content/hooks/receiveContent.ts

```diff
@@ -41,6 +42,9 @@ const hookBody: ActionHookBody<typeof receivePage | typeof locationChange> = (se
   const loadingBook = select.loadingBook(state);
   const loadingPage = select.loadingPage(state);
   const currentPath = pathname(state);
+  const queryParamsObj = query(state);
```

Comment:
It would be fine to inline the expression in the next line rather than creating this variable.